src/utils/todo/task-filing/content-loader.ts

```diff
@@ -161,6 +161,11 @@ export const addSubtitleOrContent = (todoItem: TodoItemI): void => {
     case FilingStatusE.CHANGE_REQUESTED:
       todoItem.content = TodoContentE.CHANGE_REQUESTED
       break
+    case FilingStatusE.APPROVED:
+      if (todoItem.name === FilingTypes.CONTINUATION_IN) {
+        todoItem.content = TodoContentE.APPROVED_CONTINUATION_IN
+      }
+      break
```

Comment:
For continuation-in filing, the APPROVED status does not mean the process is complete. It is just the continuation authorization is approved. So it is still a To-do item. Clicking the "Resume" button will allow user to complete the last step of the continuation-in application.